### PR TITLE
moving diffAnalyzer to a new file

### PR DESCRIPTION
**Pull request by shireenf-ibm**
#264

### CODE REVIEW

File: pkg/netpol/diff/diff_analyzer.go
Comment (adisos) [inline pkg/netpol/diff/diff_analyzer.go:1]: I would keep only lines 19-86 in the separate file for `diff_analyzer.go`, and the rest I think should remain in the original `diff.go` file.

Comment (shireenf-ibm) [reply pkg/netpol/diff/diff_analyzer.go:1]: done

Comment (shireenf-ibm) [reply pkg/netpol/diff/diff_analyzer.go:1]: shouldn't add also other exported funcs , like `NewDiffAnalyzer, and ConnectivityDiffToString`?

Comment (adisos) [reply pkg/netpol/diff/diff_analyzer.go:1]: I meant lines 19-86 from the file diff_analyzer.go of your previous commit...
I think it should include the type `DiffAnalyzer` itself, and all the DiffAnalyzer options , including also `NewDiffAnalyzer`.
The rest (exported methods and non-exported methods and functions) in the original `diff.go` file.

Comment (shireenf-ibm) [reply pkg/netpol/diff/diff_analyzer.go:1]: done

**Review by adisos — APPROVED** [review]

**Pull request merged**